workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -11,5 +11,27 @@ public class SpatialOSComponent : MonoBehaviour
         public Entity Entity;
         public World World;
         public WorkerSystem Worker;
+
+        public bool TryGetGameObjectForEntity(EntityId entityId, out GameObject linkedGameObject)
+        {
+            linkedGameObject = default(GameObject);
+            if (!Worker.TryGetEntity(entityId, out var entity))
+            {
+                Worker.LogDispatcher.HandleLog(LogType.Warning, new LogEvent("Could not find ECS entity for given SpatialOS entity id")
+                    .WithField("EntityId", entityId.Id));
+                return false;
+            }
+
+            var entityManager = World.GetOrCreateManager<EntityManager>();
```

Comment:
We can cache this on `Awake()` (or `Start()`) I think

Reply:
But it should be set once that lifecycle hits right?

Either way can do 

`private EntityManager entityManager;` <- class level

`entityManager = entityManager ?? World.GetOrCreateManager<EntityManager>();`

So at least we skip the heavier call after the first one

Reply:
Correct. `World` is not available at the time the MonoBehaviour is created, e.g. in `OnEnable()`. How about we add an Init() method for the `SpatialOSComponent` that does the one-time initialisation?